src/main/java/io/github/eisop/website/EisopSiteGenerator.java

```diff
@@ -305,6 +308,11 @@ public static void main(String[] args) throws IOException {
 
         System.out.println("Latest release: " + String.valueOf(latestRelease));
 
+        // Copy CFLogo.png to cf/
+        File cfLogo = new File(String.valueOf(latestRelease) + "/tutorial/CFLogo.png");
```

Comment:
Why does this PR need to add file `CFLogo.png`? Doesn't this take the existing logo from the latest release?
Or is that logo needed at `https://eisop.github.io/`?

Reply:
I think I had initially wanted to upload the .png file, but decided it was better to take it from the latest release in case you want to change the logo at some point.
Will remove the file.